ui/components/src/DateTimeRangePicker/TimeRangeSelector.tsx

```diff
@@ -19,6 +19,16 @@ import { formatAbsoluteRange } from './utils';
 
 const DATE_TIME_FORMAT = 'yyyy-MM-dd HH:mm:ss';
 
+function heightToPaddingY(height: string): string {
```

Comment:
It looks like you assume this is a string with a number and `px`, but I could see someone passing in other units (e.g. `rem`, `em`), which could break this functionality. I wonder if there's a slightly different way to write the CSS to better handle this -- let me poke at it a teeny bit to see if I can come up with a suggestion.

Alternatively (and much easier), maybe the prop can have a name (e.g. `heightPx` or some jsdoc over it to communicate that it has to be in pixels.

Reply:
See follow-up comment later down for a suggestion to adjust.